README.md

```diff
@@ -136,7 +136,7 @@ The following is an example event payload received by subscribers when a user en
 
 ## Space members
 
-The `members` namespace within a Space is a client-side filtered listener optimized for building avatar stacks. Subscribe to members entering, leaving, being removed from the Space (after a timeout) or updating their profile information.
+The `members` namespace contains methods dedicated to building avatar stacks. Subscribe to members entering, leaving, being removed from the Space (after a timeout) or updating their profile information.
```

Comment:
I wasn't a big fan of the wording here - I think it's not accurate and could lead to confusion (a listener is for an example a function passed to an event listener; that's not what the namespace is, just the `subscribe` method. I think also saying `client-side filtered` is quite unspecific, we should be more specific or leave out the implementation detail here, like I did above)